Homework7/Homework7/Services/SweetService.cs

```diff
@@ -0,0 +1,180 @@
+﻿using Homework7.Entities;
+using Homework7.Repositories;
+using Homework7.Models;
+
+namespace Homework7.Services
+{
+    public class SweetService : ISearch
```

Comment:
Better to name Service as ISweetService and add all the required methods in there